### DISCUSSION TITLE
Add download checksums

### DISCUSSION THREAD

**Discussion by maburdett** [glpi-project/glpi-agent#501] [Ideas]
### Your idea

On the GLPI Agent download page it would be nice to have a HSA256 or other checksum.

**Comment by g-bougard** [comment]
Hi @maburdett 

thank you for your suggestion. I think sha256 can be find in the build log for most packages, but you're right this is just missing.

We will try to add them, eventually by adding dedicated `.sha256` files and a link to that file next to download links. As there's still a lot of download links, I don't want to make the page too heavy to read.

**Reply by g-bougard** [reply]
Hi @maburdett 

just for your information, I manually added sha256 sums file on previous releases. But next nightly & normal releases will have the file generated automatically thanks to 16c6c1bdb616867ff0c3d095636ff1ddab8de467.
